### DISCUSSION TITLE
Web Analytics "Awaiting Data" Issue on Vercel Deployment

### DISCUSSION THREAD

**Discussion by iMythms** [vercel/community#4043] [Help]
Hello,

I've successfully deployed my Next.js project with TypeScript on Vercel. I followed the documentation to enable Web Analytics and all the logs indicate that the setup was successful. However, my analytics dashboard on Vercel keeps showing "Awaiting Data" even after several visits to the website.

**Comment by amyegan** [comment]
If you share a link to the site, I can look for more info about this 🙏

**Reply by iMythms** [reply]
Hey, sorry for the late reply, here is the link: mytham.me
